packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md

```diff
@@ -3,7 +3,7 @@ title: Add and configure packages
 ---
 Next, you need to add Okta to your application by installing and configuring our SDK.
 
-You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/configure-packages/) guide.
+You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/main/) guide.
```

Comment:
```suggestion
You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/-/main/#add-and-configure-packages) guide.
```

This breaks the left-nav and breadcrumbs, but at least it's jumping to the right section with the selected language preserved.